motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -532,18 +554,19 @@ void JointTrajectoryAction::controllerStateCB(
     return;
   }
 
-  if (!industrial_utils::isSimilar(all_joint_names_, msg->joint_names))
+  // joint torso_joint_b2 is not in msg->joint_names, so this code is commented out.
```

Comment:
There are four groups (not MoveGroup) for SDA dual arm robot (arm_left, arm_right, torso_1, torso_2), and torso_1 and torso_2 share a same joint (torso_joint).
for **_all_joint_names__**, there are 16 joints (7 left + 7 right + torso_joint_b1 + torso_joint_b2)
while **_msg->joint_names_** only have 15 joints (7 left + 7 right + torso_joint_b1), torso_joint_b2 is missing.
so **_industrial_utils::isSimilar()_** always return false in this scene.